tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -45,6 +45,13 @@ arp/test_unknown_mac.py:
     conditions:
       - "asic_type in ['cisco-8000']"
 
+arp/test_wr_arp.py:
+  skip:
+    reason: "Warm reboot is not supported on dualtor topology."
```

Comment:
Correction - warm reboot is supported on dualtor. The feature is broken on dualtor.